integration-tests/suites/duplicate_endpoints.go

```diff
@@ -93,27 +68,16 @@ func (s *DuplicateEndpointsTestSuite) SetupSuite() {
 	_, err = s.execContainer("socat", command)
 
 	time.Sleep(20 * time.Second)
-
-	err = s.collector.TearDown()
-	s.Require().NoError(err)
-
-	s.db, err = s.collector.BoltDB()
-	s.Require().NoError(err)
 }
 
 func (s *DuplicateEndpointsTestSuite) TearDownSuite() {
+	s.StopCollector()
 	s.cleanupContainer([]string{"socat", "collector"})
 	stats := s.GetContainerStats()
 	s.PrintContainerStats(stats)
 	s.WritePerfResults("DuplicateEndpoints", stats, s.metrics)
 }
 
 func (s *DuplicateEndpointsTestSuite) TestDuplicateEndpoints() {
-	processes, err := s.GetProcesses(s.serverContainer)
-	s.Require().NoError(err)
-	endpoints, err := s.GetEndpoints(s.serverContainer)
-	s.Require().NoError(err)
-
-	assert.Equal(s.T(), 2, len(endpoints))
-	assert.Equal(s.T(), 10, len(processes))
```

Comment:
It is a good idea to still check the number of expected processes.

Reply:
Agreed. See this thread for the current issue affecting process reporting in some of these tests: https://redhat-internal.slack.com/archives/CFMQ5C2TT/p1694699857526889